src/Event/EventManager.php

```diff
@@ -107,15 +107,15 @@ public function on($eventKey, $options = [], ?callable $callable = null)
         $argCount = func_num_args();
         if ($argCount === 2) {
             $this->_listeners[$eventKey][static::$defaultPriority][] = [
-                'callable' => $options,
+                'callable' => $options(...),
```

Comment:
We use `==` to compare closures now that all 'callable' keys are stored as closures